modules/programs/vicinae.nix

```diff
@@ -158,6 +164,10 @@ in
         assertion = cfg.systemd.enable -> cfg.package != null;
         message = "{option}programs.vicinae.systemd.enable requires non null {option}programs.vicinae.package";
       }
+      {
+        assertion = !cfg.useLayerShell -> !versionPost0_17;
+        message = ''After version 0.17, if you want to explicitly disable the use of layer shell, you need to set {option}.programs.vicinae.settings.launcher_window.layer_shell.enabled = false.'';
+      }
```

Comment:
Last thing that might be nice is having the pre-17 test trigger this assertion and assert that it is thrown

Reply:
How could that be checked in a test? I'm not very familiar with home-manager tests (I just use the asserts from https://git.sr.ht/~rycee/nmt/tree/master/item/bash-lib/assertions.sh)